apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -986,11 +1014,16 @@ const PoolCandidatesTable = ({
           component: (
             <DownloadUsersDocButton
               inTable
-              disabled={downloadingDoc}
-              onClick={
-                hasSelectedRows ? handleDocDownload : handleNoRowsSelected
+              disabled={
+                !hasSelectedRows ||
+                downloadingDoc ||
+                downloadingSingleDoc ||
+                downloadingAsyncFile
+              }
+              onClick={handleDocDownload}
```

Comment:
Rather than handle three cases, copied how it works in the users table where the download is disabled with no rows selected. Consistency?!